test/test_torch.py

```diff
@@ -260,6 +260,9 @@ def _test_math(self, torchfn, mathfn, input=None):
             input = []
             input.append(list(range(-5, 5)))
             input.append([x + 1e-6 for x in range(-5, 5)])
+            # Some vectorized implementations don't support large ranges
+            input.append([x + 1e10 for x in range(-5, 5)])
+            input.append([x - 1e10 for x in range(-5, 5)])
```

Comment:
Aren't we going with SLEEF for large inputs anyway, since the precision at this scale is too low?